bp-esa-frontend/src/components/layout/navbar.js

```diff
@@ -5,22 +5,38 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faUser } from '@fortawesome/free-solid-svg-icons'
 
 class NavBar extends Component {
-  render() {
-    return(
-        <Row className="navbar">
-            <Col sm={2}><div className="logo"><img src={logo} alt="logo" /></div></Col>
-            <Col sm={6}>
-                {/* This should dynamically change according to the present active page. */}
-                <h1>[Application] Report</h1>
-            </Col>
-            <Col sm={2}>
-                <div className="user_image">
-                    <FontAwesomeIcon icon={faUser} size='4x'/>
-                </div>
-            </Col>
-            <Col sm={2}><button>LOGOUT</button></Col>
-        </Row>
-    );
-  }  
+    pageName = () => {
+        const pathName = window.location.pathname;
+        let pageTitle;
+        if (pathName === '/reports') {
+            pageTitle = '[APPLICATION] REPORT';
```

Comment:
Here `[APPLICATION]` was meant to be a placeholder for the different application related pages that are being displayed (that is, Freckle, Slack and Email). The different pages have different table headers. See - https://docs.google.com/document/d/1qsfYvKNvmR8JJbSiAVellyNjrAwLDuqfxmhlvM3q2n4/edit

Reply:
I had not seen the link above; I will update accordingly